src/msw/printdlg.cpp

```diff
@@ -489,6 +526,48 @@ void wxWindowsPrintNativeData::InitializeDevMode(const wxString& printerName, Wi
         }
     }
 
+    // try to switch devmode to user or system default
+    if (m_devMode)
+    {
+        GlobalPtrLock lockDevMode(m_devMode);
+        LPDEVMODE tempDevMode = static_cast<LPDEVMODE>(lockDevMode.Get());
+        if (tempDevMode)
+        {
+            wxString _printerName = tempDevMode->dmDeviceName;
```

Comment:
We should avoid variable names starting with underscore.

Reply:
Ok, fixed with my latest commit.